src/schema/feeds.ts

```diff
@@ -909,20 +909,17 @@ export const resolvers: IResolvers<any, Context> = traceResolvers({
       (ctx, { author }: AuthorFeedArgs, builder, alias) =>
         builder
           .addSelect(
-            `count(*) filter (where ${alias}."scoutId" = '${author}')`,
+            `CASE WHEN ${alias}."scoutId" = '${author}' THEN 1 ELSE 0 END`,
             'isScout',
           )
           .addSelect(
-            `count(*) filter (where ${alias}."authorId" = '${author}')`,
+            `CASE WHEN ${alias}."authorId" = '${author}' THEN 1 ELSE 0 END`,
```

Comment:
Had to convert it to something simpler, the View seems to include a lot of its properties without telling, so we will have to include those columns in the `group by` but there were too many of them so I have tried moving away from using group by.

Reply:
This is not good. The group by is here to make sure we don't get duplicates. Let's keep the query as it is and try to fix it in another way

Reply:
Oh, but there is no duplicate post in the post table, right? This is not an aggregated query (only became one due to the count + filter) so there's no need for us to group it. These selects were only to identify whether the user is the author and/or the scout.

Reply:
Ok you're right!